plugins/woocommerce-admin/docs/features/onboarding.md

```diff
@@ -72,20 +72,20 @@ Both of these endpoints use WooCommerce Core's `WC_Helper_API` directly. The mai
 
 To disconnect from WooCommerce.com, go to `WooCommerce > Extensions > WooCommerce.com Subscriptions > Connected to WooCommerce.com > Disconnect`.
 
-## Jetpack Connection
+## WordPress.com Connection
```

Comment:
Naming in line with https://github.com/Automattic/woocommerce-services/pull/2677#issue-1871547685